### DISCUSSION TITLE
Add a taxonomy of communities, for browsing

### DISCUSSION THREAD

**Discussion by metasj** [knowledgefutures/pubpub#2120] [Ideas]
Often I want to see a list of communities like something I have in mind: conferences, books, preprints, reviews, overlays, art, classes, blogs, journal clubs, &c. Or I want to see similar communities by topic -- both to learn from them and to cross-pollinate.

FR: Add a taxonomy or folksonomy of communities: editable at least by community admins, and possibly by cross-site curators / admins.

Potential uses:  
* For exploring + browsing
* For statistics + trend tracking
* As a measure of where tutorials are wanted or needed (mix of new-creation, activity, rate of growth to sustained effort)
* For a filter on global activity (recent activity within one category)
* For a feed of new communities (so experienced community curators can find newcomers and reach out)

**Comment by kyledenlinger** [comment]
Yes, love this idea. I love that PubPub invites and facilitates collaboration, but each community seems to be its own little ship adrift on a vast ocean. A community portal would be a great supplement to (or perhaps a prerequisite to) my earlier suggestion for [more social following features](https://github.com/pubpub/pubpub/discussions/2118).
